tools/buildtools/checksummer.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: © 2024 ZeldaRET
+# SPDX-License-Identifier: MIT
+
+from __future__ import annotations
+
+import argparse
+import ipl3checksum
+from pathlib import Path
+import struct
+
+
+def round_up(n: int, shift: int) -> int:
+    mod = 1 << shift
+    return (n + mod - 1) >> shift << shift
+
+
+def pad_rom(rom_bytes: bytearray):
+    rom_len = len(rom_bytes)
+    fill_00 = round_up(rom_len, 12)
+    fill_FF = round_up(fill_00, 17)
+
+    rom_bytes.extend([0x00] * (fill_00 - rom_len))
+    rom_bytes.extend([0xFF] * (fill_FF - fill_00))
+
+
+def update_checksum(rom_bytes: bytearray):
+    # Detect CIC
+    # cicKind = ipl3checksum.detectCIC(rom_bytes)
+    # if cicKind is None:
+    #     eprint("Not able to detect CIC, defaulting to 6105")
+    #     cicKind = ipl3checksum.CICKind.CIC_X105
```

Comment:
Dead cod? Since we don't detect it would it better to be configurable as a function arg with an optional cli arg?